terraform/groups/ecs-service/variables.tf

```diff
@@ -51,10 +51,6 @@ variable "log_level" {
   type        = string
   description = "The log level for services to use: trace, debug, info or error"
 }
-variable "restricted_word_web_version" {
```

Comment:
Please add this back in. It is needed by the pipeline which passes the release version referenced in main.tf  to the ecs-service module

Reply:
Added back, presumably we do not need the port variable.